### PR TITLE
Enable AL Go for System Application

### PR DESCRIPTION
**Pull request by mazhelez**
- Enable [AL Go](https://github.com/microsoft/AL-Go) for System Application

### CODE REVIEW

File: .github/AL-Go-Settings.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+﻿{
+    "type":  "PTE",
+    "templateUrl":  "https://github.com/freddydk/AL-Go-PTE@main",
```
Comment (mazhelez) [inline .github/AL-Go-Settings.json:3]: Maybe `microsoft` here ;)

**Comment by mazhelez** [comment]
Enable AL-Go for everything under _Modules_

File: .github/AL-Go-Settings.json (hunk 1 above)
Comment (mazhelez) [inline .github/AL-Go-Settings.json:3]:
```suggestion
    "templateUrl":  "https://github.com/microsoft/AL-Go-PTE@main",
```

File: .github/workflows/AddExistingAppOrTestApp.yaml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    runs-on: [ bc-build ]
+    outputs:
+      telemetryScopeJson: ${{ steps.init.outputs.telemetryScopeJson }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Initialize the workflow
+        id: init
+        uses: freddydk/AL-Go-Actions/WorkflowInitialize@main
```
Comment (mazhelez) [inline .github/workflows/AddExistingAppOrTestApp.yaml:37]:
```suggestion
        uses: microsoft/AL-Go-Actions/WorkflowInitialize@main
```

File: .github/workflows/AddExistingAppOrTestApp.yaml [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+        with:
+          eventId: "DO0090"
+
+  Update:
+    runs-on: [ bc-build ]
+    needs: [ Initialization ]
+    steps:
+      - name: Add existing app
+        uses: freddydk/AL-Go-Actions/AddExistingApp@main
```
Comment (mazhelez) [inline .github/workflows/AddExistingAppOrTestApp.yaml:46]:
```suggestion
        uses: microsoft/AL-Go-Actions/AddExistingApp@main
```

File: .github/workflows/AddExistingAppOrTestApp.yaml [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+    if: always()
+    runs-on: [ bc-build ]
+    needs: [ Initialization, Update ]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Finalize the workflow
+        id: PostProcess
+        uses: freddydk/AL-Go-Actions/WorkflowPostProcess@main
```
Comment (mazhelez) [inline .github/workflows/AddExistingAppOrTestApp.yaml:63]:
```suggestion
        uses: microsoft/AL-Go-Actions/WorkflowPostProcess@main
```

**Comment by mazhelez** [comment]
> Enable AL-Go for everything module under _Modules_ in order to build and test the modules separately.

Will add later.

**Review by JesperSchulz — APPROVED** [review]

**Pull request merged**